### DISCUSSION TITLE
Initially setup BTCPay Hot Wallet with Ledger Nano S, now can't send transfers

### DISCUSSION THREAD

**Discussion by SireWilliamson** [btcpayserver/btcpayserver#5654] [Technical Support]
I am trying to transfer a balance to a crypto exchange. 
When I setup the hot wallet I used BTCPay to create an internal wallet using a full node. I didn't import another. At the time (and now) I am using a Ledger Nano S wallet for cold wallet.
When I try to make a transfer, I need to sign.
If I use my nano S I get
![image](https://github.com/btcpayserver/btcpayserver/assets/77017380/da2f219a-6ada-41ea-b3f8-d5c90da7eae2)
I read that Ledger Nano S is no longer supported.
So how do I sign my transactions now, if using my seed is not recommended?

**Comment by SireWilliamson** [comment]
From what I understand, since hot wallet is on, I shouldn't have to do any of this. Why is BTCPay not signing the transaction?
![image](https://github.com/btcpayserver/btcpayserver/assets/77017380/355a5a25-a345-44dd-97d7-2f09afef84a0)

**Reply by dennisreimann** [reply]
The problem mentioned in the initial screenshot isn't that Ledger isn't supported, but you have to open the Bitcoin app on the hardware wallet for the Vault app to communicate with the Ledger.

> Why is BTCPay not signing the transaction?

I can only guess, but it seems like you set up a hot wallet with the key from the Ledger and didn't choose the "Import keys to RPC" option. That would at least be one case in which the hardware wallet is required for signing.
